components/explore/filters/PriceFilter.vue

```diff
@@ -168,4 +171,12 @@ const toggleInputFocused = (): void => {
     box-shadow: 0 0 0 1px theme('k-blue');
   }
 }
+.padding-left {
+  padding-left: $fluid-container-padding;
+}
```

Comment:
can we rename this to something more meaningfull?

Reply:
renamed to fluid-padding